File: programmers/brute_force/sangwoo/최소직사각형.js
function solution(sizes) {
    var answer = 0;
    var listx=[],listy=[];
    for(var i =0;i<sizes.length;i++)
{       if(sizes[i][0] >= sizes[i][1])
        {   
        listx.push(sizes[i][0]);
        listy.push(sizes[i][1]);
        }
        else if(sizes[i][0] < sizes[i][1])
        {
        listx.push(sizes[i][1]);
        listy.push(sizes[i][0]);
        }          
            
}
    answer = Math.max(...listx)*Math.max(...listy);
    return answer;
}

File: programmers/brute_force/wonhyeok/최소직사각형.py
def solution(sizes):
    wallet_w, wallet_h = 0, 0
    
    for w, h in sizes:
        if h > w:
            w, h = h, w
        
        if wallet_w < w:
            wallet_w = w
        if wallet_h < h:
            wallet_h = h
        
    return wallet_w * wallet_h